cmd/state-offline-installer/runInstall.go

```diff
@@ -107,6 +94,15 @@ func runInstall(out output.Outputer, params *Params) error {
 
 	out.Print(fmt.Sprintf("Stage 1 of 3 Finished: Decompressing assets into: %s", assetsPath))
 
+	fmt.Println(licenseFileAssetPath)
+	accepted, err := prompts.PromptOfflineLicense(out, prompter, licenseFileAssetPath)
```

Comment:
I suspect this should be a TOS and not a license prompt (never seen a license prompt). But we can get the UX pass surface that issue.